ch.tigerjython.TigerJython2.yml

```diff
@@ -0,0 +1,49 @@
+id: ch.tigerjython.TigerJython2
+branch: '2.39'
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk/install.sh
+  - name: tigerjython2
+    buildsystem: simple
+    build-commands:
+      - install -Ddm755 /app/tigerjython/Lib
+      - install -Ddm755 /app/tigerjython/TestSamples
+      - sed -i "s|/usr/bin/java|/app/jre/bin/java|g" tigerjython
+      - sed -i "s|/opt/tigerjython|/app/tigerjython|g" tigerjython
+      - install -Dm644 Lib/* /app/tigerjython/Lib
+      - install -Dm644 TestSamples/* /app/tigerjython/TestSamples
+      - install -m755 tigerjython2.jar /app/tigerjython/tigerjython2.jar
+      - install -m755 tigerjython /app/tigerjython/tigerjython
```

Comment:
Symlink to /app/bin

Reply:
or just install it in `/app/bin` since this is a wrapper script.